storage/repository/parse_test.go

```diff
@@ -20,9 +20,9 @@ func TestParseURI(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "file:///tmp/foo.txt", uri.String())
 
-	uri, err = ParseURI("file://C:/tmp/foo.txt")
```

Comment:
You've stated that legacy URI will still work - so please leave the old test in so we can ensure it stays working.